erpnext/education/doctype/program_enrollment/program_enrollment.py

```diff
@@ -61,15 +61,9 @@ def validate_duplication(self):
 			frappe.throw(_("Student is already enrolled."))
 
 	def update_student_joining_date(self):
-		table = frappe.qb.DocType('Program Enrollment')
-		date = (
-			frappe.qb.from_(table)
-				.select(Min(table.enrollment_date).as_('enrollment_date'))
-				.where(table.student == self.student)
-		).run(as_dict=True)
-
-		if date:
-			frappe.db.set_value("Student", self.student, "joining_date", date[0].enrollment_date)
+		date = frappe.db.sql("select min(enrollment_date) from `tabProgram Enrollment` where student= %s", self.student)
+		((date,),) = frappe.db.sql("select min(enrollment_date) from `tabProgram Enrollment` where student= %s", self.student)
```

Comment:
- l64 and l65 queries are same but repeated? 
- using raw SQL where it was already using QB is not okay. We are trying to make ERPNext db agnostic. 


Also, the code seems to do exact same thing. Instead of the dictionary result, it's destructuring the tuple.